src/elements/content-sidebar/DetailsSidebar.scss

```diff
@@ -13,12 +13,14 @@
             margin-bottom: 0;
         }
     }
+}
 
-    .bcs-details-content {
-        @include bcs-SidebarSection;
-    }
+.bcs-DetailsSidebar-notices {
+    @include bcs-SidebarSection;
+}
 
-    .bcs-details-content:empty {
-        margin: 0;
-    }
+.bcs-DetailsSidebar-versions {
+    @include bcs-SidebarSection;
```

Comment:
Seems like this probably shouldn't be a sidebar section at all? Looks like it just needs some bottom padding.